tests/unit/test_instance_cmds.py

```diff
@@ -1057,14 +1051,16 @@
       'test': 'innows'},
      SIMPLE_MOCK_FILE, FAIL],
 
+    # Apparently the mock prompt does not work when processing stdin
+    # an exception is apparently returned
```

Comment:
I don't understand how the comment above relates to the change in this testcase?

Reply:
Modified comment to clarify and created an issue for fixing or modifying this test.